src/assets/styles/mixins.less

```diff
@@ -0,0 +1,15 @@
+﻿.right-alignment(@icon-order) {
+  right: (15 + (@icon-order * 30));
+}
+
+.svg-position {
```

Comment:
I like your suggestion @WPFerg - it feels like a better description of what this actually does.